repose-aggregator/docs/src/asciibinder/developer/docker.adoc

```diff
@@ -23,25 +23,50 @@ They can be passed in as build properties or the easier way is to add them to yo
 
 Then try to build a quick *Repose Docker* image from the root of the *Repose* project.
 
-[source,bash,subs="verbatim,attributes"]
+[source,bash]
 ----
 ./gradlew \
-    :repose-aggregator:artifacts:docker:buildUbuntuImageLocal \
-    -Prepose-version={project-version}
+    :repose-aggregator:artifacts:docker:tagUbuntuImageVersion \ # <1>
+    -x removeUbuntuImage # <2>
```

Comment:
We need to add a note in here that this works for Repose v9.0.0.0 and forward due to the package name changing from `repose-valve` to just `repose`.

Reply:
Fair point.

Reply:
Considering that, I am much more tempted to drop support for building a Docker image running a specific version of Repose. I almost did so on my first pass, but ended up leaving it in for the sake of convenience. However, there is no guarantee that future `Dockerfiles` will support installing past versions (as was the case here). Your best bet would be to check out the tag for the version you want to build an image for. If that's the approach you take, though, then there is not much point in maintaining the version selection feature here.

Still, it offers a modicum of convenience for certain cases. Hm. I guess I will leave it for now unless anyone else has an opinion on it one way or the other.